sandbox/registration-processor-mz.properties

```diff
@@ -552,6 +552,10 @@ mosip.regproc.cmd-validator.device.disable-trust-validation=false
 #maximum duration permissible  betwwen digital id timestamp and packet creation time
 mosip.regproc.cmd-validator.device.allowed-digital-id-timestamp-variation=30
 
+mosip.regproc.cmd.center.validation.processes=CORRECTION,NEW,UPDATE,LOST,BIOMETRIC_CORRECTION,DOCUMENT_CORRECTION,DEMOGRAPHIC_CORRECTION
```

Comment:
1.Change all the properties to naming convention 
2. replace it to mosip.registration.processor.validate-machine properties 
3. Add proper comments for all 3 properties that clearly explains how changing the value of the property will affect the behavior of the stage

Reply:
I have changed the properties as requested.